workloads/kube-burner-ocp-wrapper/metrics-profiles/mc-metrics.yml

```diff
@@ -4,10 +4,10 @@
 
 # ControlPlane Containers & pod metrics
 
-- query: (sum(irate(container_cpu_usage_seconds_total{name!="",container!="POD",namespace=~".+{{.HCP_NAMESPACE}}"}[2m]) * 100) by (container, pod, namespace, node)) > 0
+- query: (sum(irate(container_cpu_usage_seconds_total{name!="",container!="POD"}[2m]) * 100) by (container, pod, namespace, node)) > 0
   metricName: mgmt-containerCPU
 
-- query: sum(container_memory_rss{name!="",container!="POD",namespace=~".+{{.HCP_NAMESPACE}}"}) by (container, pod, namespace, node)
+- query: sum(container_memory_rss{name!="",container!="POD"}) by (container, pod, namespace, node)
```

Comment:
These 2 are duped, you can remove the original ones, which are not using any namespace label

Reply:
As discussed, we need etcd, kube-api, kube-controller, during the time of cluster creation, we will re-use this profile for other purposes too.